.buildkite/scripts/run_models.sh

```diff
@@ -19,4 +19,6 @@ dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
 dbt test --target "$db"
+dbt run --vars '{tiktok_ads__using_ad_country_report: false, tiktok_ads__using_campaign_country_report: false}' --target "$db" --full-refresh
```

Comment:
Looks like `tiktok_ads__using_ad_country_report` is no longer needed. 

```suggestion
dbt run --vars '{tiktok_ads__using_campaign_country_report: false}' --target "$db" --full-refresh
```